.github/workflows/spelling.yml

```diff
@@ -0,0 +1,36 @@
+name: Spell checking
+on:
+  pull_request_target:
+  push:
+  issue_comment:
+
+jobs:
+  build:
+    name: Spell checking
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout-merge
+      if: "contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2
+      with:
+        ref: refs/pull/${{github.event.pull_request.number}}/merge
+    - name: checkout
+      if: "!contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2
+    - uses: check-spelling/check-spelling@prerelease
+      with:
+        experimental_apply_changes_via_bot: 1
+        extra_dictionaries:
+          cspell:cpp/cpp.txt
+          cspell:golang/go.txt
+          cspell:node/node.txt
+          cspell:php/php.txt
```

Comment:
I'm not actually sure _why_ this dictionary was suggested. The order in which they're suggested is based on the number of items they cover, but note that if two dictionaries cover the same words, both get credit for them.

1. I'm considering having the tool list the covered words (in the logs), but it can be literally thousands of items, so I'm still not certain I want to do it.
2. I'm considering whether the tool should credit lower scoring dictionaries for words present in higher scoring dictionaries. If users care enough, they can of course apply one dictionary at a time to see how many words they're getting.